src/main/java/com/kmush12/CurrencyConverter/Converter/CurrencyConverterController.java

```diff
@@ -20,8 +18,7 @@ public CurrencyConverterController(CurrencyConverterService currencyConverterSer
         this.currencyConverterService = currencyConverterService;
     }
 
-
-    @GetMapping
+    @GetMapping("convert")
     @ResponseBody
```

Comment:
`@ResponseBody` zawiera się już w adnotacji `@RestController` (jak sobie do niej wejdziesz to zobaczysz), więc nie jest wymagana na żadnej metodzie.